iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractLoadingLimits.java

```diff
@@ -85,6 +91,66 @@ public L setPermanentLimit(double permanentLimit) {
         return (L) this;
     }
 
+    @Override
+    public L setTemporaryLimitValue(int acceptableDuration, double temporaryLimitValue) {
```

Comment:
I don't know if that's actually necessary to verify if it is > 0 because, it has to be, since `isTemporaryLimitValueValid` already checks if the argument is valid. 

But I wondered, if the new value is actually a NaN, should I throw an exception, or should I just log a WARN ?

Reply:
To be consistent, we should adopt the same behavior as the creation via the adder (i.e. the content of `AbstractLoadingLimitsAdder.TemporaryLimitAdderImpl`). It throws an exception when the value is `< 0`  or `NaN`.
So it would be better to throw an exception in these 2 cases.

Note that no exception is thrown when `isTemporaryLimitValueValid` is `false`, so relying on it for the `> 0` check won't be sufficient.